package.json

```diff
@@ -76,7 +76,7 @@
   "size-limit": [
     {
       "path": "dist/index.js",
-      "limit": "150 KB"
+      "limit": "250 KB"
```

Comment:
We've been trying to limit the size of this library to the minimum. Did you get an error that you were over the limit? Because my testing says you should be fine:
<img width="598" alt="Screen Shot 2021-06-21 at 2 44 37 PM" src="https://user-images.githubusercontent.com/41120953/122812289-805acf00-d29f-11eb-8a67-ae0a1678aa99.png">

Reply:
it said it was over by about 75kb

Reply:
plus we will have to increase the limit sooner or later in the future